packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -851,6 +851,112 @@ void main() {
     expect(tester.getTopLeft(find.text('Item 0')).dx, 72);
     expect(tester.getTopLeft(find.text('Item 1')).dx, 72);
   });
+
+  testWidgets('test tooltip', (WidgetTester tester) async {
```

Comment:
It's better to use a more descriptive test name so that future readers understand exactly what this tests for:
```suggestion
  testWidgets('PopupMenuButton tooltip defaults', (WidgetTester tester) async {
```